server/service/src/sync/translations/stock_line.rs

```diff
@@ -204,20 +206,14 @@ mod tests {
     #[actix_rt::test]
     async fn test_stock_line_push_merged() {
         // The item_links_merged function will merge ALL items into item_a, so all stock_lines should have an item_id of "item_a" regardless of their original item_id.
-        let (_, connection, _, _) = setup_all(
+        let (mock_data, connection, _, _) = setup_all(
             "test_stock_line_push_item_link_merged",
-            MockDataInserts::none()
-                .units()
-                .items()
-                .item_links_merged()
-                .names()
-                .stores()
-                .locations()
-                .barcodes()
-                .stock_lines(),
+            MockDataInserts::all(),
```

Comment:
Was there a reason we needed to go to all? Individually probably negligible on test run time, but if doing `all` for ALL tests then it might slow things down a bit?

Reply:
Sqlite has a migration cache for all(). So all() in sqlite is actually faster than individual selected data... For postgres all() is slower though.

However, I think for sqlite we should update the test code to use all() even if you selected specific data (and explicitely opt out if the test doesn't work with all())...

Reply:
I wonder if you could get a similar benefit from using postgres template databases? https://www.postgresql.org/docs/current/manage-ag-templatedbs.html

Reply:
I tried this (there is a branch somewhere) but I only got something like a 50% speed up and `slow / 2 `  was still `slow` for which reason I didn't follow through with it. Thought having a full test without cache might be more beneficial in this case.

Think the reason is that even with a template Postgres still has to populate the internal data structs while for sqlite we just do a very fast sequential copy of the database file.

Reply:
Maybe I should give this a try:
https://aiven.io/community/forum/t/faq-why-is-database-creation-from-templates-slower-for-large-databases-in-postgresql-15/421

Reply:
I am in favour to keep them. If this has been done for performance reasons, I would rather make the test caching smarter...